src/lager_udp_backend.erl

```diff
@@ -54,7 +49,7 @@ check_config(_) -> true.
 init(Params)->
     Level        = proplists:get_value(level, Params, debug),
     Formatter    = proplists:get_value(formatter,Params,lager_default_formatter),
-    FormatConfig = proplists:get_value(format_config,Params,[]),
+    FormatterConfig = proplists:get_value(formatter_config,Params,[]),
```

Comment:
A space after a `,` improves readability IMO. I know it was like this in this module, I suggest to modify at least the lines you touched.

Reply:
UDP is my next target for improvement, I'll tackle that in a separate PR 🙂